webserver/src/test/java/org/jboss/test/arquillian/ce/webserver/WebserverTestBase.java

```diff
@@ -56,15 +79,61 @@ protected static WebArchive getDeploymentInternal() throws Exception {
     }
 
     /*
+    * Allow httpClient to use untrusted connections
+    * The code below was tested with httpclient 4.3.6-redhat-1
+    * code example from; http://literatejava.com/networks/ignore-ssl-certificate-errors-apache-httpclient-4-4/
+    */
+    public static HttpClient acceptUntrustedConnClient() throws Exception {
```

Comment:
I tried to add this to ce-arq Tools, but it is using an older version of httpClient leading the tests to fail with CNF.